app/views/team/show.scala

```diff
@@ -74,8 +74,11 @@ object show:
               t.publicLeaders.nonEmpty option p(
                 teamLeaders.pluralSame(t.publicLeaders.size),
                 ": ",
-                fragList(t.publicLeaders.toList.map: l =>
-                  userIdLink(l.some))
+                fragList(
+                  t.publicLeaders.toList.map: l =>
+                    userIdLink(l.some),
+                  " "
+                )
```

Comment:
I would say option B (using CSS). And if there's no separator you can just do this:

```suggestion
                t.publicLeaders.toList.map: l =>
                  userIdLink(l.some)
```

Reply:
Probably don’t need the tolist anymore to map too

Reply:
> Probably don’t need the tolist anymore to map too

oh `t.publicLeaders` is a List already